apps/backend/src/transaction/entities/payment-method.entity.ts

```diff
@@ -10,6 +12,9 @@ export class PaymentMethodEntity {
   @Column('varchar', { length: 2, nullable: true })
   sbc_code?: string;
 
+  @Column({ enum: FileTypes, nullable: true })
```

Comment:
payment method master is agnostic of the file types. This is a master defined by PCC. I am not sure if we need the file types here...